tests/generic_config_updater/test_syslog.py

```diff
@@ -116,10 +116,13 @@ def setup_env(duthosts, rand_one_dut_hostname, cfg_facts, init_syslog_config, or
     duthost = duthosts[rand_one_dut_hostname]
     create_checkpoint(duthost)
 
-    syslog_config_cleanup(duthost, cfg_facts)
-
-    if init_syslog_config == CONFIG_ADD_DEFAULT:
+    if init_syslog_config == CONFIG_CLEANUP:
```

Comment:
>CONFIG_CLEANUP [](http://example.com/codeflow?start=29&length=14)

There are lots of code changes not listed in PR description. Could you list all the motivations?

We would like to understand if any is refactoring, or fixing something.

Reply:
Updated PR motivation. Please have a look. Thanks